pkg/agent/status_manager.go

```diff
@@ -149,3 +167,15 @@ func (mgr *Manager) updateClusterStatus(ctx context.Context, namespace, clusterI
 		klog.WarningDepth(2, "failed to update status of ManagedCluster: %v", lastError)
 	}
 }
+
+func (mgr *Manager) getNewManagedClusterLabels() map[string]string {
+	originLabels := mgr.managedCluster.GetLabels()
+	modifiedLabels := map[string]string{}
+	for k, v := range originLabels {
+		modifiedLabels[k] = v
+		if strings.HasPrefix(k, known.NodeLabelsKeyPrefix) {
```

Comment:
We've already applied this check in `getCommonNodeLabels`.

Reply:
done.